samples/dnn/optical_flow.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+'''
+This sample using FlowNet v2 model to calculate optical flow.
+
+You can download the converted .caffemodel model from https://drive.google.com/open?id=16qvE9VNmU39NttpZwZs81Ga8VYQJDaWZ
+or convert .h5 model to .caffemodel yourself.
+Download .prototxt from https://drive.google.com/open?id=19bo6SWU2p8ZKvjXqMKiCPdK8mghwDy9b
+or modify original Flownetv2.prototxt - change ChannelNorm layers to series of layers as in link above.
```

Comment:
Please rephrase so user can understand which are required steps and that are optional. Other wise on can donwload .caffemodel then "To get original model". I'd like to put everything in one place and say "Download this and that. Otherwise download original model from here".

Changes required in prototxt file I'd like to recommend to write right in the prototxt file as a reference to origin prototxt file.